common/static/js/capa/src/formula_equation_preview.js

```diff
@@ -139,8 +139,7 @@ formulaEquationPreview.enable = function () {
                         if (inputData.jax) {
                             // Set the text as the latex code, and then update the MathJax.
                             MathJax.Hub.Queue(
-                                ['Text', inputData.jax, latex],
-                                ['Reprocess', inputData.jax]
+                                ['Text', inputData.jax, latex]
```

Comment:
Note to reviewers: I found that just setting the Text is sufficient for updating the display, and also calling Reprocess was leading to an extra repaint/flash.